### PR TITLE
A test case for selecting values from attributes (XML only)

### PR DESCRIPTION
**Pull request by chrdebru**

### CODE REVIEW

File: test-cases/RMLTC0023a-XML/mapping.ttl
Comment (DylanVanAssche) [inline test-cases/RMLTC0023a-XML/mapping.ttl:9]: We should fix the blank node here.

**Review by DylanVanAssche — CHANGES_REQUESTED** [review]

File: test-cases/RMLTC0023a-XML/mapping.ttl
Comment (chrdebru) [reply test-cases/RMLTC0023a-XML/mapping.ttl]: Done!

**Review by DylanVanAssche — APPROVED** [review]

**Comment by DylanVanAssche** [comment]
Will leave it a bit open for the maintainers before merging it.

CC: @pmaria @andimou

**Pull request merged**
